contents/blog/brand.md

```diff
@@ -0,0 +1,152 @@
+
+We introduced it because the company was getting larger, and the scope of what we were setting out to achieve had expanded. We now have 8+ products, and we produce a ton of content to help engineers build better products. 
+
+The need for total clarity wasn't as important later on, relative to the need for us standing for something. It was necessary to start small then widen.
+
+## You need to create your brand for someone...
+
+One of the most important things we did was to [create an Ideal Customer Profile](/newsletter/ideal-customer-profile-framework).
+
+When we launched, we weren't sure quite how focused we'd need to be on engineers vs product managers. We realized, after launching, that we could go all in on engineers and technical founders, who make up around 80% of our users. Our brand and product has become more oriented towards these users as a result.
+
+Here are some specific ways we got more specific for this audience after we clarified it:
+
+* We added API examples to our website.
+
+* We added a "you'll hate PostHog if" section to our homepage, which calls out many pet hates that engineers have when trying to buy, install and use more traditionally-targeted software that they are made to implement.
+
+* We chose a more flexible pricing model – it's far more complex, but it's [much better value](/blog/analytics-pricing). We felt more technical people would be happier than non-technical people coping with more complexity to get a better deal. We want developers to be able to manage their spend with us across our eight products, not a "single price".
+
+* In the product, we worked on [SQL access](/docs/hogql), so technical people could get to the underlying data to debug things.
+
+* We [redesigned our entire UX](/blog/posthog-as-a-dev-tool) to have higher information density, dark mode and more flexibility.
+
+## ...but your brand must be a reflection of _you_
+
+We have often been impressed at the level of polish that others can achieve.
+
+Quoting Cory, our lead designer: "We aren't the best in the world at being polished, but we can be the best in the world at being ourselves".
+
+We sometimes make business decisions based on who we are, and what we want to do – not always what may, for example, generate the most revenue growth.
+
+For example, we set out initially expecting to have an open core model. We'd offer open-source software, and some people in big enterprises, would self-host and pay for the party for everyone else. 
+
+We felt this would work as there was no real competition, whereas in cloud we had lots of competitors with over $100M in annual revenue. However, we simply love shipping new features, and after our first hackathon, we wound up with session replay (i.e. a new feature big enough to be its own product), which took off, and this resulted in a multi-product approach. 
```

Comment:
```suggestion
We felt this would work as there was no real competition, whereas in cloud we had lots of competitors with over $100m in annual revenue. However, we simply love shipping new features, and after our first hackathon, we wound up with session replay (i.e. a new feature big enough to be its own product), which took off, and this resulted in a multi-product approach. 
```